Solutions/Cyware/Playbooks/Send_Incident_To_Cyware_Orchestrate/README.md

```diff
@@ -0,0 +1,77 @@
+# Send Incident to Cyware Orchestrate
+
+This Azure Logic App template is designed to integrate Azure Sentinel incidents with Cyware Orchestrate, enabling automated incident response actions. When an Azure Sentinel alert triggers, this Logic App automatically sends the incident details to Cyware Orchestrate.
```

Comment:
Hi @AashiqRamachandran,

Please replace `Azure Sentinel` text with `Microsoft Sentinel` in all occurrences. Thanks